src/strings.bif

```diff
@@ -1107,6 +1107,30 @@ function find_last%(str: string, re: pattern%) : string
 	return zeek::val_mgr->EmptyString();
 	%}
 
+## Finds the first occurrence of a pattern in a string.
+##
+## str: The string to inspect.
+##
+## re: The pattern to look for in *str*.
+##
+## Returns: The first string in *str* that matches *re*, or the empty string.
+##
+## .. zeek:see:: find_all find_all_ordered find_last strstr
+function find_first%(str: string, re: pattern%) : string
```

Comment:
I think it would be nice if there's an optional third parameter giving the offset where to start looking (and then add the same to `find_last`)

Reply:
I'll make a ticket to do this separately.